src/Umbraco.Web/HealthCheck/Checks/Config/TrySkipIisCustomErrorsCheck.cs

```diff
@@ -38,7 +38,7 @@ public override IEnumerable<AcceptableConfiguration> Values
         {
             get
             {
-                var recommendedValue = _serverVersion >= new Version("7.5.0")
+                var recommendedValue = _serverVersion >= new Version("7.5")
```

Comment:
@nul800sebastiaan The change was to update the check to just 7.5 and not 7.5.0 as it was originally, this was the reason why we seen the wrong result on our 7.5 IIS instance since 7.5.0 > 7.5 when the version comparison is made here.

For the translation I think the correct values are being sent, its just the wrong one was being used. In the translation file it says:
 
` <!-- The following keys get these tokens passed in:
	  0: Current value
	  1: Recommended value
	  2: Server version
   -->`

Which all seem to be correct and needed but it was just using the recommended value in part of the translation that was talking about the current IIS version  eg:

> Try Skip IIS Custom Errors is set to true and you're using IIS version true

with my change it should now read 

> Try Skip IIS Custom Errors is set to true and you're using IIS version 7.5